dbt/adapters/mindsdb/relation.py

```diff
@@ -1,4 +1,4 @@
-from dataclasses import dataclass
+from dataclasses import dataclass,field
```

Comment:
need a space between 'dataclass, field'

Reply:
not really related in dbt api but just make the materialize a little more to the typical one.

Especially for the return value, a real Relation instead of null value. Also add pre-hook/post-hook

Reply:
thanks fore review and agreed